### PR TITLE
Production update 25.3.2024

### PR DESCRIPTION
**Pull request by villejuutila**

### CODE REVIEW

File: src/file-upload/handler.ts [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+/**
+ * Lambda function that returns a presigned URL for a PUT request to upload a file to the specified Amazon S3 bucket
+ */
+export const main: APIGatewayProxyHandler = async ({ body }) => {
+  try {
+    const { path, contentType } = JSON.parse(body || '{}');
+    const { BUCKET_NAME, BUCKET_REGION } = process.env;
+
+    if (!path) {
+      throw Error("Invalid request body");
```
Comment (Jdallos) [inline src/file-upload/handler.ts]: Could it be worth setting a status code for the catch here?

File: src/file-upload/handler.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    }
+
+    if (!BUCKET_NAME || !BUCKET_REGION) {
+      throw Error("Invalid environment variables");
```
Comment (Jdallos) [inline src/file-upload/handler.ts]: Same suggestion as above

**Review by Jdallos — APPROVED** [review]

File: src/file-upload/handler.ts (hunk 1 above)
Comment (villejuutila) [reply src/file-upload/handler.ts]: Resolved.

File: src/file-upload/handler.ts (hunk 2 above)
Comment (villejuutila) [reply src/file-upload/handler.ts]: Resolved.

**Pull request merged**